modules/dotnet-cli.test.js

```diff
@@ -2,20 +2,52 @@
 // #region Imports
 // -----------------------------------------------------------------------------------------
 
-const child_process = require("child_process");
 const dotnetBuild = require("./dotnet-build");
 const dotnetCli = require("./dotnet-cli");
 const dotnetPath = require("./dotnet-path");
 const faker = require("faker");
 const shell = require("shelljs");
+const testUtils = require("../tests/test-utils");
 
 // #endregion Imports
 
+// -----------------------------------------------------------------------------------------
+// #region Mocks
+// -----------------------------------------------------------------------------------------
+
+// Mocking dir module to suppress lots of extra output from popd/pushd errors from lack of actual
+// directory stack.
+jest.mock("./dir");
+
+// #endregion Mocks
+
 // -----------------------------------------------------------------------------------------
 // #region Tests
 // -----------------------------------------------------------------------------------------
 
 describe("dotnetCli", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region cmd
+    // -----------------------------------------------------------------------------------------
+
+    describe("cmd", () => {
+        test("given a string array of arguments, it returns a properly command string to the Cli executable", () => {
```

Comment:
I know it will expand the description a bit, but I would argue that we should describe exactly what we are expecting here instead of using properly.